jsonrpc.go

```diff
@@ -250,7 +276,7 @@ func (client *RPCClient) SetHTTPClient(httpClient *http.Client) {
 	client.httpClient = httpClient
 }
 
-func (client *RPCClient) newRequest(notification bool, method string, params ...interface{}) (*http.Request, error) {
```

Comment:
Now you cannot do something like call("myMethod", "myParam1", 123, "myParam3") anymore, can you?